api/graphql/schema.graphql

```diff
@@ -1777,6 +1787,14 @@ type Mutation {
     id: UUID!
     assessmentStep: AssessmentStepInput @spread
   ): AssessmentStep @update @guard @can(ability: "update", find: "id")
+  swapAssessmentStepOrder(stepA: UUID!, stepB: UUID!): [AssessmentStep]
```

Comment:
I'd suggest more precise names for these parameters.  Maybe stepIdA and stepIdB?

Reply:
Updated as part of [Update swap assessmentStepOrder with feedback](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8456/commits/7f95c7631827cbe7847acb3a0573f5fb43d92681).